File: yangxj96-bean/src/main/java/io/github/yangxj96/bean/user/Authority.java
package io.github.yangxj96.bean.user;

import com.baomidou.mybatisplus.annotation.TableField;
import com.baomidou.mybatisplus.annotation.TableName;
import io.github.yangxj96.common.base.BasicEntity;
import lombok.*;
import lombok.experimental.SuperBuilder;

import java.io.Serial;
import java.io.Serializable;

/**
 * 权限表实体
 *
 * @author yangxj96
 */
@Data
@EqualsAndHashCode(callSuper = true)
@ToString
@SuperBuilder
@AllArgsConstructor
@NoArgsConstructor
@TableName(value = "db_user.t_authority")
public class Authority extends BasicEntity implements Serializable {

    @Serial
    private static final long serialVersionUID = 1L;

    /**
     * 名称
     */
    @TableField(value = "\"name\"")
    private String name;

    /**
     * 描述
     */
    @TableField(value = "description")
    private String description;

}

File: yangxj96-bean/src/main/java/io/github/yangxj96/bean/user/Authority.kt
package io.github.yangxj96.bean.user

import com.baomidou.mybatisplus.annotation.TableField
import com.baomidou.mybatisplus.annotation.TableName
import io.github.yangxj96.common.base.BasicEntity
import java.io.Serializable

/**
 * 权限表实体
 */
@TableName(value = "db_user.t_authority")
class Authority : BasicEntity(), Serializable {

    /**
     * 名称
     */
    @TableField(value = "code")
    var code: String? = null

    /**
     * 描述
     */
    @TableField(value = "description")
    var description: String? = null
}